openc3/lib/openc3/models/gem_model.rb

```diff
@@ -36,7 +36,7 @@ module OpenC3
   # and destroy to allow interaction with gem files from the PluginModel and
   # the GemsController.
   class GemModel
-    include Api
+    extend Api
```

Comment:
Doesn't extend add as class methods vs include adds as instance methods

Reply:
Yep.  That was the bug as we were using get_setting in a class method not an instance method.